server.go

```diff
@@ -90,7 +91,11 @@ func (a *App) handleFromLocalItem(ctx context.Context, msg Message, dst int) err
 	if err != nil {
 		return err
 	}
-
+	err = update.Sign(a.identity)
+	if err != nil {
+		log.Error().Err(err).Msg("failed to sign")
```

Comment:
if you return an error, the log is not needed, errors can be logged when they can't be propagated anymore